src/components/EventCard/styles.js

```diff
@@ -47,6 +53,14 @@ const styles = (theme) => ({
     fontSize: '1.05rem',
     textAlign: 'left',
     whiteSpace: 'nowrap',
+    [theme.breakpoints.down('xs')]: {
```

Comment:
you should be able to add the `noWrap` prop on Typography to handle ellipsizing it.
https://material-ui.com/api/typography/

Reply:
this is weird cause it already has ellipsis on the class. ya try using the material ui prop.

Reply:
Oh, its intentionally to show all the title and not ellipsis it. TT want when it's on mobile size(only one card per row) we show full title like first pic, if its on larger screen(2 or 3 cards per row), we ellipsis the title.